context-free.md

```diff
@@ -1,10 +1,10 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+People who want to generate art/images.
```

Comment:
Adding to Jean, there are many languages out there to create art, is there a specific subset of people that this restrictive DSL is meant for?